docs/upgrade-testing.md

```diff
@@ -136,6 +136,9 @@ spec:
         name: knative-serving
     modelmeshserving:
       managementState: Managed
+    modelregistry:
+      managementState: Managed
+      registriesNamespace: "odh-model-registries"
```

Comment:
```suggestion
      managementState: Removed
      registriesNamespace: "rhoai-model-registries"
```

Reply:
this Managed -> Removed doc changes probably not necessary?

Reply:
ok. Fine for me to change, but I wanted to keep difference with ODH as less as possible.